Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -1187,6 +1192,95 @@ void MenuBar::ImportWiiSave()
   }
 }
 
+void MenuBar::ImportWiiSaves()
+{
+  QString folder =
+      DolphinFileDialog::getExistingDirectory(this, tr("Select Save Folder"), QDir::currentPath());
+
+  if (folder.isEmpty())
+    return;
+
+  QDirIterator it(folder, QStringList(QStringLiteral("*.bin")), QDir::Files,
+                  QDirIterator::Subdirectories);
+  QStringList failure_details;
+  size_t success_count = 0;
+  size_t fail_count = 0;
+  bool yes_all = false;
+  bool no_all = false;
+
+  while (it.hasNext())
+  {
+    const QString file = it.next();
+
+    auto can_overwrite = [&] {
+      if (yes_all)
+        return true;
+      if (no_all)
+        return false;
+
+      auto response = ModalMessageBox::question(
+          this, tr("Save Import"),
+          file + QStringLiteral(": ") +
+              tr("Save data for this title already exists in the NAND. Consider backing up "
+                 "the current data before overwriting.\n\nOverwrite existing save data?"),
```

Comment:
Some languages may not want to use ": " exactly. You should instead do something like:

```suggestion
              tr("%1: Save data for this title already exists in the NAND. Consider backing up "
                 "the current data before overwriting.\n\nOverwrite existing save data?").arg(file),
```

Same for the ones below.